packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.controller.js

```diff
@@ -24,4 +35,11 @@ export default class SoftphoneController {
     }
     this.osName = MOBILE_OS.windows;
   }
+
+  deleteDevice(deviceId) {
+    this.$state.go(
+      'telecom.telephony.billingAccount.line.dashboard.softphone.delete',
+      { deviceId },
+    );
+  }
```

Comment:
Add a `goToDeleteDevice` function in the routing. Dont inject `$state` into controller.